modules/hmmer/hmmbuild/main.nf

```diff
@@ -0,0 +1,35 @@
+process HMMER_HMMBUILD {
+    tag '$meta.id'
+    label 'process_low'
+
+    conda (params.enable_conda ? "bioconda::hmmer=3.3.2" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/hmmer:3.3.2--h87f3376_2':
+        'quay.io/biocontainers/hmmer:3.3.2--h1b792b2_1' }"
+
+    input:
+    tuple val(meta), path(alignment)
```

Comment:
missing optoinal inputs of:

```
  --mx <s>      : substitution score matrix (built-in matrices, with --singlemx)
  --mxfile <f>  : read substitution score matrix from file <f> (with --singlemx)
```

Reply:
I never used them... Can't they be left to `$args`?

Reply:
Difficult with the file perhaps...

Reply:
Exactly, they won't be staged, so we need to accomodate _all_ possible input files (even if optional)

Reply:
There are many other options that are not explicitly covered, but could be left to `$args`. I think single sequence input is a very rare use case. 
Or do you perhaps mean only the `--mxfile <f>` that requires an input file? That would perhaps be difficult to have in `$args`? Should I prepare for input via `meta`?

Reply:
Oops, sorry yes - I meant just `--mxfile`! No, should not come via meta, should be an explicit input chanel - but if you don't need it can be just filled with an empty list `[]`.

Reply:
I base my request of all 'possible' input files based on what is reported in `--help` of that particular tool, so I would just focus on those.

Reply:
Done.